solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -1298,6 +1298,16 @@ public static ModifiableSolrParams params(String... params) {
     return Utils.makeMap(params);
   }
 
+  public static SolrParams mapParams(String... vals) {
```

Comment:
Please let's not pollute SolrTestCaseJ4 with this needless method.  STCJ4 is out of control and I feel the need to be a policeman.  See the existing `params` method.  The addition of javabin randomly is weird.  I understand you are just trying to move this module and you didn't write this code... so move it somewhere else or consider if it's maybe not useful to randomly alter the response format.

Reply:
Yea I want to make this change outside of this PR. `mapParams` is being used across classes for no good reason. I agree it needs to move somewhere and its a weird method. I'll see where else in `test-framework` makes sense.